samples/sample-rdk-rules/config-rules/SOME_OTHER_RULE/parameters.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Our JSON files use 4 space indentation, could you fix that in the JSON files in this PR?
See https://github.com/awslabs/aws-deployment-framework/blob/9c383d08ac7de4308aa95ba43210d14aaba9849b/.editorconfig